elvis.config

```diff
@@ -3,14 +3,28 @@
   elvis,
   [
    {config,
-    [#{dirs => ["apps/*/src", "apps/*/test", "test/ejabberd_tests/tests"],
+    [#{dirs => ["apps/*/src"],
        filter => "*.erl",
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
 		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
 		]
-      }
+      },
+     #{dirs => ["apps/*/test", "test/ejabberd_tests/tests"],
+       filter => "*.erl",
+       ruleset => erl_files,
+       rules => [{elvis_style, line_length, #{limit => 100,
+                                              skip_comments => false}},
+                 {elvis_style, dont_repeat_yourself, disable},
+                 {elvis_style, invalid_dynamic_call, disable},
+                 {elvis_style, no_debug_call, disable},
+                 {elvis_style, god_modules, disable},
+                 {elvis_style, no_spec_with_records, disable},
+                 {elvis_style, used_ignored_variable, disable},
+                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z]*)$"}}
```

Comment:
Because otherwise Elvis doesn't allow usage of ?assertMatch/Equal macros, because those after expanding result in usage of variables like __X and __V. There is no other workaround I could think of that actually works. Those variables are named this way not without a reason.